backend/src/main/java/me/performancereservation/api/RefundController.java

```diff
@@ -0,0 +1,72 @@
+package me.performancereservation.api;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import me.performancereservation.domain.refund.RefundService;
+import me.performancereservation.domain.refund.dto.RefundDetailResponse;
+import me.performancereservation.domain.refund.dto.RefundRequest;
+import me.performancereservation.domain.refund.enums.RefundStatus;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/refunds")
+public class RefundController {
+
+    private final RefundService refundService;
+
+    /*--- USER 요청에 대응 ---*/
+    // 본인 id와 일치하는 모든 환불내역 리스트 반환
+    @GetMapping("/user/{userId}")
+    public ResponseEntity<List<RefundDetailResponse>> getAllRefundDetailsWithUserId(@PathVariable Long userId) {
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(refundService.findAllRefundsDetailByUserId(userId));
+    }
+
+    // 특정 예약id 환불요청 -> 환불내역 생성
+    @PostMapping("/user/new-refund")
+    public ResponseEntity<Long> createNewRefund(@RequestBody RefundRequest refundRequest) {
```

Comment:
```suggestion
    public ResponseEntity<CreateRefundResponse> createNewRefund(@RequestBody RefundRequest refundRequest) {
```

리소스를 생성하고 난 뒤에는 생성한 리소스를 그대로 내려주는 것이 '만약에' 프론트엔드에서 생성한 환불정보를 확인용도로 표시해주는 페이지가 있다고 쳤을 때 refundId로 조회 요청을 새롭게 보내지 않아도 돼서 좋은 것 같습니다!

Reply:
아하.. 그러면 이미 있는 RefundDetailResponse로 응답을 보내는 건 어떤가요? 쿼리 조회가 필요하긴 하겠지만.. 공연 제목을 보려면 불가피할 것 같기도 해서요

Reply:
아 보니까 그래도 될 것 같습니다 👍

Reply:
RefundId로 RefundDetailResponse를 조회하는 메서드를 추가해서 컨트롤러에서 이 dto를 반환하는 방향으로 고쳐보겠습니다!